code/modules/mob/living/living.dm

```diff
@@ -852,7 +852,7 @@
 
 	// These should be tracked by status effects
 	losebreath = 0
-	set_disgust(0)
+	set_disgust_effect(0)
```

Comment:
Not needed anymore, handled via `remove_on_fullheal` on the status effect

Reply:
Fuck, true.